docs/userguide.md

```diff
@@ -282,44 +282,17 @@ successfully verified a working federation-v2 deployment.
 To cleanup the example simply delete the namespace and its placement:
 
 ```bash
-kubectl delete federatednamespaceplacement test-namespace
 kubectl delete ns test-namespace
+kubectl delete federatednamespaceplacement test-namespace
 ```
 
 ## Cleanup
 
-### Unjoin Clusters
```

Comment:
I prefer that we add `unjoin` back when https://github.com/kubernetes-sigs/federation-v2/pull/144 got merged, this can map to the above `join` command.

Reply:
I opted to move the unjoin operation to the `delete-federation.sh` script because it was a bunch of bash commands and makes it cleaner. When we support `kubefed unjoin` we can add it back here and simplify the script.

Reply:
Yes, same thinking as you.